subcommands/config/updates.go

```diff
@@ -24,7 +24,15 @@ in a device group. When run without options, prints out the current configuratio
   fioctl config updates --group beta --apps shellhttpd
 
   # Set the docker apps and the tag for devices:
-  fioctl config updates --group beta --apps shellhttpd --tag master`,
+  fioctl config updates --group beta --apps shellhttpd --tag master
+
+  # Set the docker apps to none for devices, this will make the device run no apps:
+  fioctl config updates --apps ,
+
+  # Set the docker apps to inherit from their parent, meaning:
+  # If for a device apps are not set  - all apps are installed
+  # If for a device apps are set      - those apps are installed
```

Comment:
We still aren't explain what the "parent" will be. I'm not sure parent is a good word. What we are trying to describe is how are config files are processed. We use the same logic as [systemd](https://www.freedesktop.org/software/systemd/man/latest/systemd-system.conf.html#Configuration%20Directories%20and%20Precedence) where we walk through:
 * /usr/lib/sota/conf.d/
 * /var/sota/sota.toml
 * /etc/sota/conf.d/

We need a concise way to explain this.

And that `,` and `-` are so subtle here that we should probably call out that they are special characters.

Reply:
IIUC the group level is the top one here (as we allow no factory-wide updates config).
So, probably, we may put here a sentence like this:
```suggestion
# Set the docker apps to a system default (all apps on most devices):
```

A _system default_ seems to be rather specific to me, without diving into internal details (which can differ from device to device).